superset/assets/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -28,172 +51,265 @@ const propTypes = {
 const defaultProps = {
   animation: true,
   onChange: () => {},
-  value: '',
+  value: 'Last week',
 };
 
+
 export default class DateFilterControl extends React.Component {
   constructor(props) {
     super(props);
-    const value = props.value || '';
+    const value = props.value || defaultProps.value;
     this.state = {
+      type: TYPES[0],
+
+      // for range
       num: '7',
-      grain: 'days',
-      rel: 'ago',
-      dttm: '',
-      type: 'free',
-      free: '',
+      grain: TIME_GRAIN_OPTIONS[3],
+      rel: RELATIVE_TIME_OPTIONS[0],
+      common: COMMON_TIME_FRAMES[0],
+
+      // for start:end(includes freeform)
+      since: DEFAULT_SINCE,
+      until: DEFAULT_UNTIL,
+      isFreeform: {},
```

Comment:
I feel like this value should be a boolean if its prefixed with `is`?

Reply:
What if we initialize it as:

```js
isFreeform: { since: false, until: false }
```

I can also rename it to `freeformFields`.